test/integrationtests/LorisIntegrationTest.class.inc

```diff
@@ -45,10 +45,12 @@ abstract class LorisIntegrationTest extends PHPUnit_Framework_TestCase
     public function setUp()
     {
         // Set up database wrapper and config
-        $this->config = NDB_Config::singleton(__DIR__ . "/../../project/config.xml");
-        $database     = $this->config->getSetting('database');
+        $factory      = NDB_Factory::singleton();
+        $this->config = $factory->Config(CONFIG_XML);
```

Comment:
can we maybe put this in an if defined instead of assuming it's defined, so that it's still possible to write tests that don't use the bootstrap file?